server/routes.js

```diff
@@ -180,7 +180,7 @@ function withConfigPartial(
       const client = getClient(req.hostname);
       return client
         .getConfig()
-        .then((config) => configWrapper(config, domainSlug))
+        .then((config) => configWrapper(config, domainSlug, { req }))
```

Comment:
configWrapper needs `req` to get the current (req.url)
current URL is needed because we need to pass it as query param to the new pagbuilder redirects API